staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -311,13 +311,16 @@ func (req *request) NoteQueued(inQueue bool) {
 	}
 }
 
-func (req *request) Finish(execFn func()) bool {
+func (req *request) Finish(execFn func()) (idle bool) {
```

Comment:
If I understand correctly, this is a preference rather than an absolue prohibition.

Reply:
> If I understand correctly, this is a preference rather than an absolue prohibition.

a strong enough preference that this case doesn't qualify for an exception.  @tkashem no named returns please.